rplugin/python/nvim_ipy.py

```diff
@@ -119,7 +120,6 @@ def __call__(self, msg):
             self.is_active = False
 
 @neovim.plugin
-@neovim.encoding(True)
```

Comment:
why remove this? Note that decoding already is default in python3, so this ensures consistent behaviour also in python2.